src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -83,6 +83,7 @@ protected function findTemplate($template, $throw = true)
                 $template = $this->parser->parse($template);
                 $file = $this->locator->locate($template);
             } catch (\Exception $e) {
+            } catch (\Throwable $e) {
```

Comment:
There are a lot of `InvalidArgumentException` beeing thrown here and a few `RuntimeException`, but no interface contract for those. So just not add the catch Throwable then?